### PR TITLE
#RI-2432 - implement REDIS_STACK build type

### PR DESCRIPTION
**Pull request by ArtyomPodymov**
#RI-2432 - implement REDIS_STACK build type

### CODE REVIEW

**Review by arthosofteq — DISMISSED** [review]

File: redisinsight/ui/src/App.tsx [hunk 1]
```diff
@@ -1,44 +1,56 @@
...
-import { NavigationMenu, Notifications, Config, ShortcutsFlyout, MonitorConfig } from './components'
+import { Config, MonitorConfig, NavigationMenu, Notifications, ShortcutsFlyout } from './components'
 import { ThemeProvider } from './contexts/themeContext'
 import MainComponent from './components/main/MainComponent'
 
 import themeDark from './styles/themes/dark_theme/_dark_theme.lazy.scss'
 import themeLight from './styles/themes/light_theme/_light_theme.lazy.scss'
 
 import './App.scss'
+import { BuildType } from "uiSrc/constants/env";
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/App.tsx:20]: move this import after other `uiSrc/` imports. Also, please, change to single quotes

File: redisinsight/ui/src/components/consents-settings/ConsentsSettingsPopup/ConsentsSettingsPopup.tsx [hunk 2]
```diff
@@ -21,8 +25,16 @@ import lightLogo from 'uiSrc/assets/img/light_logo.svg'
 import styles from '../styles.module.scss'
 
 const ConsentsSettingsPopup = () => {
+  const history = useHistory()
+  const { server } = useSelector(appInfoSelector)
   const { theme } = useContext(ThemeContext)
 
+  const handleSubmitted = () => {
+    if (server && server.buildType === BuildType.RedisStack && server?.fixedDatabaseId) {
+      history.push(Pages.browser(server?.fixedDatabaseId))
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/consents-settings/ConsentsSettingsPopup/ConsentsSettingsPopup.tsx:34]: no need optional chaining here

File: redisinsight/ui/src/components/main-router/MainRouter.tsx [hunk 3]
```diff
@@ -1,19 +1,39 @@
 import React from 'react'
 import { Redirect, Switch } from 'react-router-dom'
-import ROUTES from 'uiSrc/constants/routes'
 import extractRouter from 'uiSrc/hoc/extractRouter.hoc'
 import { registerRouter } from 'uiSrc/services/routing'
 import RouteWithSubRoutes from 'uiSrc/utils/routerWithSubRoutes'
+import { useSelector } from 'react-redux'
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/main-router/MainRouter.tsx:6]: please structure imports

File: redisinsight/ui/src/components/consents-settings/ConsentsSettingsPopup/ConsentsSettingsPopup.tsx (hunk 2 above)
Comment (ArtyomPodymov) [reply redisinsight/ui/src/components/consents-settings/ConsentsSettingsPopup/ConsentsSettingsPopup.tsx]: done

File: redisinsight/ui/src/App.tsx (hunk 1 above)
Comment (ArtyomPodymov) [reply redisinsight/ui/src/App.tsx]: done

File: redisinsight/ui/src/components/main-router/MainRouter.tsx (hunk 3 above)
Comment (ArtyomPodymov) [reply redisinsight/ui/src/components/main-router/MainRouter.tsx]: done

**Review by romansergeenkosofteq — APPROVED** [review]

**Review by arthosofteq — APPROVED** [review]

**Pull request merged**